### PR TITLE
Add changelog and display it on Sandstorm about page

### LINKED ISSUE #440: Show changelog on about page
We should show a change log on the "about" page, summarizing what's new in each build. It would perhaps be nice to pull the text from a changelog file in the repository home directory for easy maintenance, and so that people can read it there too. This probably requires a Makefile rule which copies the changelog from the source root into the shell's `public` directory, then code for the about page which downloads that file dynamically (much like is currently done for the contributor list), but I'm open to other ideas.

### PR DESCRIPTION
**Pull request by jparyani**
Fixes #440

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -162,11 +162,14 @@ shell-env: tmp/.shell-env
 
 # Note that we need Ekam to build node_modules before we can run Meteor, hence
 # the dependency on tmp/.ekam-run.
-tmp/.shell-env: tmp/.ekam-run $(IMAGES)
+tmp/.shell-env: tmp/.ekam-run $(IMAGES) CHANGELOG.md
 	@mkdir -p tmp
 	@touch tmp/.shell-env
 	@mkdir -p node_modules/capnp
 	@bash -O extglob -c 'cp src/capnp/!(*test*).capnp node_modules/capnp'
+	@echo '<template name="changelog">' > shell/client/changelog.html
```
Comment (kentonv) [inline Makefile]: Can you write a separate make rule for shell/client/changelog.html and have tmp/.shell-env depend on it, as we do for images?

**Comment by jparyani** [comment]
Done

**Comment by kentonv** [comment]
Oh, also make sure it's deleted by `make clean`.

**Comment by jparyani** [comment]
Done

**Pull request merged**